build.gradle.kts

```
@@ -33,6 +33,9 @@ val javaMajorVersion: JavaVersion by extra(JavaVersion.VERSION_11)
 val ciBranch: String? by extra(System.getenv("CIRCLE_BRANCH"))
 val isCI: Boolean by extra(System.getenv("CI") == "true")
 val jetbrainsMarketplaceToken: String? by extra(System.getenv("JETBRAINS_MARKETPLACE_TOKEN"))
+val pluginSignPrivateKey: String? by extra(System.getenv("PRIVATE_KEY"))
```

I'd use full names for env vars:
```suggestion
val pluginSignPrivateKey: String? by extra(System.getenv("PLUGIN_SIGN_PRIVATE_KEY"))
```
and so for the 2 other ones.

So that it's clear even in a few years what's the purpose of the given env var in CI.